rocAL/rocAL_pybind/examples/rocAL_api_tf_classification_reader.py

```diff
@@ -39,6 +41,7 @@ def main():
     # Args
     imagePath = args.image_dataset_path
     rocalCPU = False if args.rocal_gpu else True
+    device = "cpu" if rocalCPU else "gpu"
     batch_size = args.batch_size
```

Comment:
Naming convention does not like it follows snake case style
rocALCPU and imagePath are in camel case but rest of them are in snake case
Please stick to one convention instead of 2